cloudformation/imagebuilder/pcluster_install.yaml

```diff
@@ -196,7 +239,7 @@ phases:
               set -x
               chef-client --local-mode --config /etc/chef/client.rb --log_level info --logfile /var/log/chef-client.log --force-formatter --no-color --chef-zero-port 8889 --json-attributes /tmp/dna.json --override-runlist aws-parallelcluster::default
 
-      - name: Bootstrap
+      - name: InstallCfnBootstrap
         action: ExecuteBash
         inputs:
           commands:
```

Comment:
use separate action for bucket url in bootstrap

Reply:
done